app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Your Recipes</h1>
+
+<button class="border-light bg-info rounded-3 my-4 p-1 text-decoration-none">
+   <%= link_to "Add New Recipe", new_recipe_path %>
+</button>
+
+<div id="recipes" >
+  <% @recipes.each do |recipe| %>
+    <div class="border border-secondary rounded-4 p-3 my-5"  id="<%= dom_id recipe %>">
+      <h2>
+        <strong>Name:</strong>
+        <%= recipe.name %>
+      </h2>
+
+      <p>
+        <strong>Description:</strong>
+        <%= recipe.description %>
+      </p>
+
+
+      <button class="border-light bg-info rounded-3 my-4 p-1 text-decoration-none">
+          <%= link_to "Show this recipe", recipe %>
+      </button>
+
+      <button class="border-light bg-info rounded-3 my-4 p-1 text-decoration-none">
+          <%= link_to "Remove", recipe %>
+      </button>
+      
+    </div>
+  <% end %>
```

Comment:
- [ ] As requested by the previous reviewer, please fix your styles because currently they are not being applied. I have seen your comments on the previous reviews when you said that we maybe need to install bootstrap. I have installed bootstrap in `gemfile` but It didn't work. However, I saw that you are using yarn in your project too. I tried installing the dependencies of yarn but I was getting errors while I am sure that I have it installed. Have you installed that bootstrap with yarn? If so, please indicate all the procedures for running your project very well in the README file so that the next reviewer will know how to run it and get the expected results.
**Screenshot**
![image](https://user-images.githubusercontent.com/90222110/202871508-6739b0f8-2648-4234-876e-06d47fd593a2.png)

Reply:
-How are you at Bertan I don't understand why bootstrap isn't working on your side I have even asked other peers to try our application and it's working well.
- We have included instructions that have worked on our peers' computers